test/any/parsing/BaseParser-test.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('BaseParser', function() {
+  beforeEach(function() {
+    foam.CLASS({
+      name: 'TestParser',
+      extends: 'org.chromium.webidl.BaseParser',
+
+      constants: {
+        PARSE_COMPLETE: 'Parse complete',
```

Comment:
These should be on `BaseParser` instead, I think. That way classes (like this one) interacting with `BaseParser` know what messages it uses; if the messages change, we don't have to update tests.